docs/content/en/docs/topics/restore-a-backup/_index.md

```diff
@@ -165,21 +192,37 @@ insert into users (email, name, state) values ('sue@help.com', 'Sue Sas', 'CA');
 insert into users (email, name, state) values ('tom@yes.com', 'Tom and Jerry', 'NV');
 ```
 
-Copy the cql file to the k8ssandra container (pod) :
+Copy the cql file to the k8ssandra container (pod):
 
 `kubectl cp test_data.cql k8ssandra-dc1-default-sts-0:/tmp -c cassandra`
 
-Add the data to the Cassandra database:
+Before you can launch the CQLSH instance that's deployed by K8ssandra in your Kubernetes cluster, you'll need authentication credentials. By default, the username secret for K8ssandra is `k8ssandra-superuser`. To verify, you can extract and decode the username secret:
+
+`kubectl get secret k8ssandra-superuser -o jsonpath="{.data.username}" | base64 --decode`
+`k8ssandra-superuser`
+
+The command output (second line above) verified that the username to use on a subsequent command is `k8ssandra-superuser`.
+
+Next, extract and decode the password secret. For example:
 
-`kubectl exec -it k8ssandra-dc1-default-sts-0 -c cassandra -- cqlsh -f /tmp/test_data.cql`
+`kubectl get secret k8ssandra-superuser -o jsonpath="{.data.password}" | base64 --decode`
+```
+7kZV7YUFSWUTksJ9nfZoWzuTL0qGSgjp54kEVAlgrbMTW_E3RXcTsg
```

Comment:
Consider adding a comment here that this value is for your demo cluster. The value will be different on their installation.

```suggestion
# Note this value is for _this_ cluster. Your cluster will have a different value.
7kZV7YUFSWUTksJ9nfZoWzuTL0qGSgjp54kEVAlgrbMTW_E3RXcTsg
```

Reply:
Inc'd edit.